site/src/Librecores/ProjectRepoBundle/Entity/ClassificationHierarchy.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+namespace Librecores\ProjectRepoBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * ClassificationHierarchy
+ *
+ * @ORM\Table(name="classification_hierarchy")
+ * @ORM\Entity(repositoryClass="Librecores\ProjectRepoBundle\Repository\ClassificationHierarchyRepository")
```

Comment:
As your repository classes are empty anyways, can't you just use the doctrine default ones?